### PR TITLE
add steamsetpoint to website

### PR DESCRIPTION
**Pull request by murmeltier08**

### CODE REVIEW

**Comment by sway** [comment]
N.B. Fixes https://github.com/rancilio-pid/clevercoffee/issues/279

File: src/rancilio-pid.cpp [hunk 1]
```diff
@@ -239,6 +239,7 @@ SysPara<double> sysParaPidKpSteam(&steamKp, PID_KP_STEAM_MIN, PID_KP_STEAM_MAX,
 SysPara<uint8_t> sysParaPidOn(&pidON, 0, 1, STO_ITEM_PID_ON);
 SysPara<uint8_t> sysParaUsePonM(&usePonM, 0, 1, STO_ITEM_PID_START_PONM);
 SysPara<uint8_t> sysParaUseBDPID(&useBDPID, 0, 1, STO_ITEM_USE_BD_PID);
+SysPara<double> sysParaSteamSetPoint(&steamSetPoint, STEAM_SETPOINT_MIN, STEAM_SETPOINT_MIN, STO_ITEM_STEAM_SETPOINT);
```
Comment (kjyv) [inline src/rancilio-pid.cpp:242]: second STEAM_SETPOINT_MIN should be STEAM_SETPOINT_MAX

File: src/Storage.cpp [hunk 2]
```diff
@@ -240,6 +242,14 @@ static inline int32_t getItemAddr(sto_item_id_t itemId, uint16_t* maxItemSize =
             size = STRUCT_MEMBER_SIZE(sto_data_t,weightsetpoint);
             break;
 
+        case STO_ITEM_STEAM_SETPOINT:
+            addr = offsetof(sto_data_t,weightsetpoint );
+            size = STRUCT_MEMBER_SIZE(sto_data_t,weightsetpoint);
```
Comment (kjyv) [inline src/Storage.cpp:247]: This should not be weightsetpoint but steamsetpoint ;)

File: src/rancilio-pid.cpp (hunk 1 above)
Comment (kjyv) [reply src/rancilio-pid.cpp:242]: Also these values are not set anywhere yet?

**Pull request merged**